tests/frame/to_native_test.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any
+
+import narwhals.stable.v1 as nw
+
+
+def test_to_native(constructor_eager: Any) -> None:
```

Comment:
It would be nice to make the test name a bit more descriptive, e.g. `test_to_native_returns_object_of_original_class_type`